### DISCUSSION TITLE
crazyflie 2.1

### DISCUSSION THREAD

**Discussion by fangjir** [bitcraze/discussions#1122] [Q&A]
i know "cflib is an API written in Python that is used to communicate with the Crazyflie and Crazyflie 2.0 quadcopters."
so crazyflie_lib_python support crazyflie 2.1?

**Comment by knmcguire** [comment]
Hi! The current version of the [crazyflie python library](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/) (aka the CFlib) is only compatible with [Crazyflie 2.0](https://www.bitcraze.io/products/old-products/crazyflie-2-0/) and [Crazyflie 2.1](https://www.bitcraze.io/products/crazyflie-2-1/) (along with the [Bolt ](https://www.bitcraze.io/products/crazyflie-bolt-1-1/) and [Flapper](https://store.bitcraze.io/products/flapper-nimble-starter-kit)). 

The Crazyflie 1 support for the python library is dropped since 2019 (I think), so you'll have to revert the library to an earlier commit or try an earlier release

**Reply by fangjir** [reply]
@knmcguire thanks
